sci-mathematics/gimps/gimps-29.8.6.ebuild

```diff
@@ -1,36 +1,46 @@
-# Copyright 1999-2018 Gentoo Foundation
+# Copyright 1999-2019 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=6
+EAPI=7
 
 inherit pax-utils systemd readme.gentoo-r1
 
 DESCRIPTION="The Great Internet Mersenne Prime Search"
 HOMEPAGE="https://www.mersenne.org/"
+
+# The author has finally caved in to the pressure of putting the build
+# number in the package name. Some dark magic necessary to get to the
+# package name
+
+MY_PV=$(ver_rs 2 'b' )
+
 SRC_URI="
-	amd64? ( ftp://mersenne.org/gimps/p95v${PV/./}.linux64.tar.gz )
-	x86? ( ftp://mersenne.org/gimps/p95v${PV/./}.linux32.tar.gz )"
+	amd64? ( ftp://mersenne.org/gimps/p95v${MY_PV/./}.linux64.tar.gz )
```

Comment:
Could you rename the distfile into more identifiable one? Add `-> ${P}.linux64.tar.gz` or similar to this and line below. 

Also don't use `git mv` but make a new commit for addition and removal.

Reply:
> Could you rename the distfile into more identifiable one? Add -> ${P}.linux64.tar.gz or similar to this and line below.

Is it really necessary? It's always been like this: p95 stands for Prime95, which was the original project name when it was a Windows only program, and it's been retained for historical reasons.

> Also don't use git mv but make a new commit for addition and removal.

I didn't.

I did a `git rm` of the old file, a `git add` of the new one and then `repoman dx commit`

I have no idea why it appeared as a `git mv`

Reply:
Ok, the distfile is good. But you need to make a new commit for new addition, then new commit for file removal.

Reply:
Done. Is it ok now?

Reply:
Thanks! FYI, you can remove empty DEPEND from the ebuild if you wish to make it cleaner.